src/transform/split-declaraion.js

```diff
@@ -0,0 +1,28 @@
+import traverser from '../traverser';
+import multiReplaceStatement from '../utils/multi-replace-statement';
+import VariableDeclaration from '../syntax/variable-declaration';
+
+export default function(ast) {
+  traverser.replace(ast, {
+    enter(node, parent) {
+      if (node.type === 'VariableDeclaration' && node.declarations.length > 1) {
+        this.skip();
```

Comment:
I'd actually like to phase out the uses of `this.skip()` and others as they are somewhat magical - if you don't know ESTraverse API well enough you'll be wondering where does this `skip()` method come from. Better to instead return `traverser.VisitorOption.Skip` after calling `splitDeclarations()`.

See: https://github.com/estools/estraverse/wiki/Usage